cmd.sh

```diff
@@ -1,6 +1,44 @@
 #!/usr/bin/env sh
 
 set -e
+# first we check if a release already exists. We can't use the GET release by tag endpoint because
+# draft releases aren't tagged using the format we expect. Instead we list all releases and filter
+# using JQ to see if a release with the provided name already exists.
+listReleasesStatusCode=$(curl \
+  --silent \
+  --user "${loginUsername}:${loginPassword}" \
+  -X GET "https://api.github.com/repos/${owner}/${repo}/releases?per_page=30" \
```

Comment:
I tried to cover that in the PR description, but it's a bit confusing. Getting a release by a tag name doesn't work for draft releases since the tag for the release is created when the release is moved out of draft mode. I ran into this issue a few times in testing but when I created a draft release my releases tag looked like `untagged-$commitish`